consensus/polybft/signer/common.go

```diff
@@ -0,0 +1,139 @@
+package bls
+
+import (
+	"crypto/rand"
+	"crypto/sha256"
+	"errors"
+	"fmt"
+	"math/big"
+
+	bn256 "github.com/umbracle/go-eth-bn256"
+)
+
+var (
+	zeroBigInt = *big.NewInt(0)
+	oneBigInt  = *big.NewInt(1)
+	g2         = *new(bn256.G2).ScalarBaseMult(&oneBigInt)  // Generator point of G2 group
+	zeroG1     = *new(bn256.G1).ScalarBaseMult(&zeroBigInt) // Zero point in G2 group
+	zeroG2     = *new(bn256.G2).ScalarBaseMult(&zeroBigInt) // Zero point in G2 group
+
+	// this is a copy of point p is a prime over which we form a basic field: 36u⁴+36u³+24u²+6u+1.
+	// TODO delete this point and use bn256.p
+	bn256P, _ = new(big.Int).SetString("21888242871839275222246405745257275088696311157297823662689037894645226208583", 10)
+)
+
+// genRandomBytes generates byte array with random data
+func genRandomBytes(size int) (blk []byte) {
```

Comment:
This method is only used in tests, please move it from this file to the test file where it's used

Reply:
Fixed through #796